examples/feature_demo/earth.py

```diff
@@ -0,0 +1,125 @@
+"""
+Earth
+=====
+
+
+This example demonstrates the effect of rendering a globe using MeshPhongMaterial.
+"""
+
+################################################################################
+# .. warning::
+#     An external model is needed to run this example.
+#
+# To run this example, you need a model from the source repo's example
+# folder. If you are running this example from a local copy of the code (dev
+# install) no further actions are needed. Otherwise, you may have to replace
+# the path below to point to the location of the model.
+
+import os
+from pathlib import Path
+
+try:
+    # modify this line if your model is located elsewhere
+    model_dir = Path(__file__).parents[1] / "data"
+except NameError:
+    # compatibility with sphinx-gallery
+    model_dir = Path(os.getcwd()).parent / "data"
+
+
+################################################################################
+# Once the path is set correctly, you can use the model as follows:
+
+# sphinx_gallery_pygfx_render = True
+
+import imageio.v3 as iio
+import numpy as np
+import pylinalg as la
+from wgpu.gui.auto import WgpuCanvas, run
+import pygfx as gfx
+
+
+# Init
+canvas = WgpuCanvas(size=(640, 480), max_fps=60, title="Earth")
+renderer = gfx.renderers.WgpuRenderer(canvas)
+
+scene = gfx.Scene()
+
+earth_geometry = gfx.sphere_geometry(63.71, 100, 50)
+
+
+def load_texture(path, flip=False):
+    img = iio.imread(path)
+    if flip:
+        img = np.ascontiguousarray(np.flipud(img))
+    tex = gfx.Texture(img, dim=2)
+    return tex
+
+
+# Earth
+earth_material = gfx.MeshPhongMaterial(shininess=10)
+earth_material.map = load_texture(
+    model_dir / "planets" / "earth_atmos_4096.jpg", flip=True
+)
+earth_material.specular_map = load_texture(
+    model_dir / "planets" / "earth_specular_2048.jpg", flip=True
+)
+earth_material.emissive = "#888866"
+earth_material.emissive_map = load_texture(
+    model_dir / "planets" / "earth_lights_2048.png", flip=True
+)
+earth_material.emissive_intensity = 1.5
+
+# # Uncomment to use light map instead of emissive map
+# earth_material.light_map = earth_material.emissive_map
+# earth_geometry.texcoords1 = earth_geometry.texcoords
+# earth_material.light_map_intensity = 2.5
+
+earth_material.normal_map = load_texture(
+    model_dir / "planets" / "earth_normal_2048.jpg", flip=True
+)
+earth_material.normal_scale = 0.85, -0.85
+
+earth = gfx.Mesh(earth_geometry, earth_material)
+earth.local.rotation = la.quat_from_euler((0, -1.0, 0), order="XYZ")
+scene.add(earth)
+
+# Clouds
+clouds_material = gfx.MeshPhongMaterial()
+clouds_map_tex = load_texture(
+    model_dir / "planets" / "earth_clouds_2048.png", flip=True
```

Comment:
This example adds over 5 MB of data to the repo. Any idea why the cloud map is especially large?

Reply:
I believe it's because the image format is PNG, and the cloud map has transparency with an alpha channel.